notebooks/lesson05_random-forest-deep-dive.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>One minor suggestion: maybe use the same kwargs for <code>RandomForestRegressor</code> explicitly (like bootstrap=True/False) so it is transparent what settings you are using and they can compare them easily.</p><p> </p><p>I don't know how clear that is to the students, maybe it's obvious. You probably have a better feeling for that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>good idea! implemented</p>